index.js

```diff
@@ -79,16 +116,17 @@ module.exports = inputOptions => {
 	};
 
 	if (process.type === 'renderer') {
+		//	Debounced because some packages, for example React, because of their error boundry feature, throws many identical uncaught errors
```

Comment:
No need for a tab here. Might want to reword to avoid this many comma splices.
```suggestion
		// Debounced because some packages, for example React, because of their error boundry feature, throws many identical uncaught errors
```

Reply:
Suggestion taken. Rewritten to:

> Debounced because some packages can throw many identical uncaught errors (eg. React's ErrorBoundary component)